pages/1.12/release-notes/1.12.2/index.md

```diff
@@ -0,0 +1,162 @@
+
+- DCOS-40878, DCOS_OSS-4164 - You can set a new configuration option to control the maximum number of active subscribers are allowed on a Master node event stream at any given time. The `--max_operator_event_stream_subscribers` option helps to prevent load balancers or proxy servers from keeping connections to the event stream endpoint open after a client disconnects. The default value for the `--max_operator_event_stream_subscribers` option is 1000 subscribers. If your network has clients that do not close connections immediately, you might want to lower the number of subscribers allowed. In addition, a new metric for operator event stream subscribers returns the total number of subscribers to the master node's operator event stream.
+
+- DCOS_OSS-4514, DCOS_OSS-4666 - This release adds service address (A or AAAA) records for public and private nodes to DNS. You can use `public.thisnode.thisdcos.directory` to return the **public IP addresses** for a local agent or master node or `thisnode.thisdcos.directory` to return the **private IP addresses** for a local agent or master node. 
+
+    For DNS lookup requests, service address `A` records convert domain names to corresponding IP addresses using IPv4 notation for a given host. Service address `AAAA` records return IP addresses using IPv6 address notation for given host.
+
+# Known Issues and Limitations
+This section covers any known issues or limitations that don’t necessarily affect all customers, but might require changes to your environment to address specific scenarios. The issues are grouped by feature, functional area, or component. Where applicable, issue descriptions include one or more issue tracking identifiers.
+
+### Marathon Plugin Dependency
+If you have custom Marathon plugins or have added any Marathon-dependent customization to your cluster, you might need to update the plugins or customized components after you upgrade to this release. For example, if you have a plugin with a dependency on Scala Logging version 3.1.0, which was compiled with Scala 2.11, you need to upgrade the Scala Logging package to version 3.7.2 compiled with Scala 2.12 to maintain compatibility with the logging library used in the Marathon package included in this release of DC/OS.
+
+### Service Account Permissions for Metics Collection
+Metrics in DC/OS, version 1.12 and newer, are based on Telegraf. Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database. The  Telegraf program runs under the service accounts `dcos_telegraf_master` and `dcos_telegraf_agent`. These two service account must be granted `dcos::superuser permissions`.
+
```

Comment:
@lsgunnlsgunn Could you check with ROQ team to see if there are any new `Certified Packages and DC/OS Versions` for this release? If yes, then you must be document it in RN, update the Version Policy page, and add the Version Policy page link in RN. 

Make sure to check with ROQ team when to expose the new versions in Version Policy page. This is generally done after 4pm (when the release is announced) on the day of a release.

Reply:
Who/What is the ROQ team? Is there a slack channel or specific person to ping?

Reply:
@gauripowale ^^